cmd/kpod/kill.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"os"
+)
+
+var (
+	killFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "signal, s",
+			Usage: "Signal to send to the container",
+			Value: "KILL",
+		},
+	}
+	killDescription = "The main process inside each container specified will be sent SIGKILL, or any signal specified with option --signal."
+	killCommand     = cli.Command{
+		Name:        "kill",
+		Usage:       "Kill one or more running containers with a specific signal",
+		Description: killDescription,
+		Flags:       killFlags,
+		Action:      killCmd,
+		ArgsUsage:   "[CONTAINER_NAME_OR_ID]",
+	}
+)
+
+// killCmd kills one or more containers with a signal
+func killCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) == 0 {
+		return errors.Errorf("specify one or more containers to kill")
+	}
+	config, err := getConfig(c)
+	if err != nil {
+		return errors.Wrapf(err, "could not get config")
+	}
+	server, err := libkpod.New(config)
+	if err != nil {
+		return errors.Wrapf(err, "could not get container server")
+	}
+	defer server.Shutdown()
```

Comment:
Just wondering if we want to do the shutdown if the update below doesn't work?  I'm not sure, but maybe we only do it after the update or if we've a --force switch?

Reply:
I think we still need it to ensure containers/storage adequately cleans up mounts and other things. Even if we fail to load other containers, we did set up a Store, and we need to shut it down cleanly.

Reply:
Yes we should always close Shutdown

Reply:
Perhaps we should move this code to libpod and stop everyone from using libkpod at this point.  @baude could you do this using 

runtime = getRuntime()

Look at kpod pull()

Otherwise @umohnani8 WIll end up having to port it.

Reply:
We still need a lot of code to land for that to be ready. It's (mostly) written, but would need reviews, CI, etc.

Reply:
We got to start getting that rolling in.  @mrunalp is the road block?

Reply:
Would like him or @runcom to review, yeah. But there's still enough code for two more PRs before we're really ready for container operations - one to populate the state implementation, another to add the OCI interaction code and basic container ops. And I haven't written the code to load in existing state from disk yet.